storage/storage_tests/storage_tests.cpp

```diff
@@ -493,11 +500,13 @@ UNIT_TEST(StorageTest_DownloadMapAndRoutingSeparately)
     }
   });
 
-  TIndex const index = storage.FindIndexByFile("Azerbaijan");
+  bool const isSingleMwm = version::IsSingleMwm(storage.GetCurrentDataVersion());
```

Comment:
Are you sure? Part of this test (downloading Map) are still valid even in case single mwm case.